### DISCUSSION TITLE
Issue with installing vscode extensions through devcontainer.json

### DISCUSSION THREAD

**Discussion by locallens07** [devcontainers/community#174] [General]
I am having an issue with devcontainer. I have a  devcontainer.json file and its not installing the vscode extensions. I am using  a custom Dockerfile

Note: Running devcontainer inside a coder workspace deployed on a kubernetes cluster.

Need help on this
[devcontainer.json](https://github.com/user-attachments/files/17746141/devcontainer.json)

**Comment by bamurtaugh** [comment]
Thanks for letting us know. Could you please provide some more details, i.e. error log message(s) and screenshot(s) or a gif of your flow?
